e2e-tests/conf/chaos-network-loss.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-loss-example
+  namespace: namespace
+spec:
+  action: loss
+  mode: one
+  loss:
+    loss: "25"
```

Comment:
why only 25? please use all confs from pxc e.g. https://github.com/percona/percona-xtradb-cluster-operator/blob/main/e2e-tests/conf/chaos-network-loss.yml

Reply:
We are using chaos-mesh version 2, pxc uses old chaos-mesh version 0.5.1

Reply:
done